### PR TITLE
`CLRequests` cleanup

### PR DESCRIPTION
**Pull request by acolytec3**
Clean-up following integration of EIP 6110, 7002, and 7685 code
 
- Moves all `CLRequest` related code to `vm/src/requests.ts`
- Adds detailed explanation of parsing Deposit event logs
- Cleans up field/function naming
- Adds prague config to test-runner from #3387 
- Add better forkTime check for EIP 2935
- Make test runner use Typescript code instead of `dist/cjs/...`
- Add optional `depositContractAddress` field to `ChainConfig` in `common`

### CODE REVIEW

File: packages/vm/src/requests.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      const validatorPubkey = slicedBytes.slice(20, 68) // 48 Bytes
+      const amount = slicedBytes.slice(68, 76) // 8 Bytes / Uint64
+      const rlpData = RLP.encode([sourceAddress, validatorPubkey, amount])
+      const request = new CLRequest(withdrawalRequestType, rlpData)
+      requests.push(request)
+    }
+  }
+}
+
+export const DEPOSIT_CONTRACT_ADDRESS = '0x00000000219ab540356cBB839Cbe05303d7705Fa'
```
Comment (g11tech) [inline packages/vm/src/requests.ts:88]: I think we should move this into Chain struct or in the chains field

**Review by holgerd77 — DISMISSED** [review]
Looks good, nice cleanup and improvements! 🙂🙏

**Review by holgerd77 — APPROVED** [review]
LGTM

**Review by g11tech — APPROVED** [review]
lgtm

**Pull request merged**